lightning/src/chain/keysinterface.rs

```diff
@@ -164,6 +164,12 @@ pub trait ChannelKeys : Send {
 	/// our counterparty may (though likely will not) close the channel on us for violating the
 	/// protocol.
 	fn sign_channel_announcement<T: secp256k1::Signing>(&self, msg: &msgs::UnsignedChannelAnnouncement, secp_ctx: &Secp256k1<T>) -> Result<Signature, ()>;
+
+	/// Set the remote funding key.  This should be done immediately on incoming channels
```

Comment:
This doc is a bit strange, the caller is *us*, so there is no should, there is only what rust-lightning does/guarantees it will do.

Reply:
fixed